roles/maas_postgres/tasks/teardown.yaml

```diff
@@ -5,12 +5,11 @@
 
 - name: Setting config dirs for uninstall
   ansible.builtin.set_fact:
-    maas_postgres_deb_config_dir: "/etc/postgresql/{{ maas_postgres_version_number }}/main/"
-    maas_postgres_snap_config_dir: "/var/snap/maas-test-db/common/postgres/data/"
+    maas_postgres_config_dir: "/etc/postgresql/{{ maas_postgres_version_number }}/main/"
 
 - name: Setting overarching config dir for uninstall
   ansible.builtin.set_fact:
-    maas_postgres_config_dir: "{{ maas_postgres_deb_config_dir if maas_postgres_installation_type|default(maas_installation_type)|lower == 'deb' else maas_postgres_snap_config_dir }}"
+    maas_postgres_config_dir: "{{ maas_postgres_config_dir }}"
```

Comment:
This seems redundant without the snap dir

Reply:
Oh yeah :laughing: one sec